### DISCUSSION TITLE
autoconfig without emailaddress parameter

### DISCUSSION THREAD

**Discussion by guillaumearnx** [modoboa/modoboa#3278] [Help]
hello, I use k9-mail as an email client and it cannot automatically detect the server settings.
According to their forum, it's because, for security reasons, the client does not send the email address as a parameter in its autoconfig request.
e.g.: instead of contacting **https://autoconfig.domain.tld/mail/config-v1.1.xml?emailaddress=mail@domain.tld**, it contacts **https://autoconfig.domain.tld/mail/config- v1.1.xml**

on my modoboa server it returns a 500 error.
Is it possible to use such a process? 
Thank you
Best regards

**Comment by tonioo** [comment]
I don't see how such request could return a result if no address email is provided... Do you know how they intend to provide the address ?

**Reply by guillaumearnx** [reply]
according to the [explanations](https://forum.k9mail.app/t/autoconfig-url-emailaddress-not-supported/8336/6), this would be a problem concerning the "privacy" of users. they do not want the email to be sent to the server if the user does not consent. actually each domain has its own autodiscover and autoconfig configurations, so saying that there would be a default setting if you don't provide an address seems strange. it's not great of them

**Comment by guillaumearnx** [comment]
Any update on this ?

**Reply by tonioo** [reply]
No, but I think that's something automx version 1 does not support and this is the version still deployed by the installer. Switching to automx2 could solve the issue (not sure about that) but it works differently from the previous version so we need to investigate a bit more
